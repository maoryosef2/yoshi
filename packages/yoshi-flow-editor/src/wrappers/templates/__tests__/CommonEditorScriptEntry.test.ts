import commonEditorScriptEntry from '../CommonEditorScriptEntry';

describe('CommonEditorScriptEntry template', () => {
  it('generates correct template with entry editorScript file', () => {
    const generateEditorScriptEntryContent = commonEditorScriptEntry({
      editorEntryFileName: 'project/src/editor.app.ts',
    });

    expect(generateEditorScriptEntryContent).toMatchSnapshot();
  });
});
