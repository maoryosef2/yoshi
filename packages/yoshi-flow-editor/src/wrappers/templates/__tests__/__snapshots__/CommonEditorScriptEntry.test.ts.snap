// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommonEditorScriptEntry template generates correct template with entry editorScript file 1`] = `
"
  export * from 'project/src/editor.app.ts';
"
`;
